forge/src/runner.rs

```diff
@@ -267,6 +267,12 @@ impl<'a, B: Backend + Clone + Send + Sync> ContractRunner<'a, B> {
 
         let (address, mut evm, init_logs) = self.new_sputnik_evm()?;
 
+        let errors_abi = if let Some(contracts) = known_contracts {
+            foundry_utils::flatten_funcs_and_events(contracts).2
+        } else {
+            self.contract.clone()
```

Comment:
Seems like you should be able to avoid this clone if you do

```rust
let errors_abi = known_contracts.map(|contracts| foundry_utils::flatten_funcs_and_events(contracts).2);
if let Some(ref abi) = errors_abi {
    abi
} else {
      self.contract // or maybe &self.contract
};

Reply:
This should have been deleted oops

Reply:
we have that in `exec_info` already

Reply:
disregard, thats in a different spot